internal/server/webserver/buildmanager.go

```diff
@@ -154,7 +154,7 @@ func Build(goos, goarch, suppliedConnectBackAdress, fingerprint, name string, sh
 		return "", err
 	}
 
-	buildArguments = append(buildArguments, fmt.Sprintf("-ldflags=-s -w -X main.destination=%s -X main.fingerprint=%s -X github.com/NHAS/reverse_ssh/internal.Version=%s", suppliedConnectBackAdress, fingerprint, f.Version))
+	buildArguments = append(buildArguments, fmt.Sprintf("-ldflags=-s -w -X main.destination=%s -X main.fingerprint=%s -X github.com/NHAS/reverse_ssh/internal.Version=%s -X github.com/NHAS/reverse_ssh/internal.server.webserver.username=%s -X github.com/NHAS/reverse_ssh/internal.server.webserver.password=%s", suppliedConnectBackAdress, fingerprint, f.Version, username, password))
```

Comment:
The additional link-time insertions will not be added to the client binary as they are targeting the internal web server? 
I dont think this will work

Reply:
This is to support each client to embed an independent username and password. I have created an independent web api, which can be called by the user passing in the username and password.

Reply:
So, effectively if anyone extracts the username and password from any client binary they'll be able to connect to a hosted RSSH server? 

Also, just reiterating, I am not moving to letting clients do password based auth to the server.

Reply:
Only the controlled end will embed the user name and password. Even if someone reverses the user name and password, it can only be connected as the controlled end. Even if it is reversed, no one should be willing to connect.